.wp-env.json

```diff
@@ -3,6 +3,7 @@
 	"plugins": [
 		"."
 	],
+	"themes": [ "WordPress/theme-experiments/twentytwenty-blocks" ],
```

Comment:
In my testing, all the themes from the repository are being loaded and not just 2021 blocks. I might have  misunderstood the `path` property in the source parser? cc @noahtallen

Reply:
@youknowriad do you have a `.wp-env.override` file which might be changing this? I think what you mentioned should only map the specific theme and nothing else :/

The alternative issue could be that `git` is downloading the entire repo and loading it to that mapped directory?

Reply:
> The alternative issue could be that git is downloading the entire repo and loading it to that mapped directory? (instead of just a path to part of the repo)

I think this is probably what's happening here but I thought the `path` property on the "source" object is meant to address that but I guess I misunderstood what's this property used for.

Reply:
I made it work with a new property.

Reply:
Cool. I'm not entirely sure, I haven't touched this portion of the code 😅 It could also just have been buggy because no one tried to use it recently